docs/core/run-time-config/garbage-collector.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Garbage collector config settings
+description: Learn about run-time settings for configuring how the garbage collector manages memory.
+ms.date: 11/13/2019
+ms.topic: reference
+---
+# Run-time configuration options for garbage collection
+
+This page contains information about garbage collector (GC) settings that can be changed at run time. If you're trying to achieve peak performance of a running app, consider using these settings. However, most users will not need to concern themselves with these settings.
+
+On this page, settings are arranged into groups of settings that you can use in conjunction with each other to achieve a specific result.
```

Comment:
```suggestion
On this page, settings are arranged into different groups that you can use in conjunction with each other to achieve a specific result.
```

Reply:
To me that makes it sound like the groups can be used in conjunction with each other. I'll try to think of different wording that doesn't duplicate settings.